### ISSUE TITLE
Appsflyer integration is not sending

### ISSUE THREAD

**Issue by DarwinVJ** [la-haus/flutter-segment#58]
Having configured the integration with appsflyer, it usually happens that events are sent with: integrations: {}, is there any reason for this to happen?

**Comment by spegoraro** [comment]
I've noticed that the `integrations` field in the raw payload has `AppsFlyer: false` for all events when I set `appsflyerIntegrationEnabled: true` in the config.

**Comment by RicharC293** [comment]
Same problem with amplitude

**Comment by aldo-mymo** [comment]
@DarwinVJ were you able to solve this issue? Having the same problem. iOS attribution works fine, Android does not.
@spegoraro I noticed the false in the integrations but same for iOS, knowing that iOS works.

**Comment by danielgomezrico** [comment]
Is this already solved on 3.13.1 ?

**Comment by Imgkl** [comment]
@danielgomezrico No. I tested this. Even on version 3.13.1, I'm seeing the integrations for appsflyer as false even with the `appsflyerIntegrationEnabled: true`

<img width="274" alt="image" src="https://github.com/la-haus/flutter-segment/assets/25414681/d7f418b7-3056-416b-9693-31178faf6e6d">
